### PR TITLE
feat: 100 days of oss

### PR DESCRIPTION
**Pull request by BekahHW**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Add #100DaysOfOSS challenge to OpenSauced community docs. The challenge is flexible and participants can join in whenever they're ready. The PR adds a new file `100-days-of-oss.md` and updates `sidebars.js`. 

_Generated using [OpenSauced](https://opensauced.ai/)._

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by bdougie — APPROVED** [review]
I really love this and will add a slide to my OSCA keynote on this as well. This will be an amazing onramp for so many new and first time contributors. 

Thanks for putting this together @BekahHW

File: docs/community/100-days-of-oss.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+To make the most of your #100DaysOfOSS journey, here are some additional resources and events you can explore:
+
+1. **Weekly Twitter Spaces:** Join our weekly Twitter Spaces sessions where we discuss open source topics, share insights, and connect with like-minded individuals. Follow us on [Twitter](https://twitter.com/saucedopen) to stay updated on upcoming sessions.
+
+2. **Community Events:** Discover a wide range of events on our community docs page. Whether it's hack days, workshops, or office hours, these events provide excellent opportunities to learn, collaborate, and find new projects to contribute to. Visit our [community docs page](https://docs.opensauced.pizza/community/welcome-to-the-community) to explore the upcoming events.
+
+3. **Weekly Contribution Opportunities:** If you're actively looking for open source projects to contribute to, check out the [weekly post](https://dev.to/bekahhw/series/23323) for new contribution opportunities.
+
+4. **Weekly Office Hours:** Have questions or need help? Join us on [Discord](https://discord.gg/dXvGd8mz) during our office hours or post in our #100DaysOfOSS channel. We're here to help you succeed!
+
```
Comment (CBID2) [inline docs/community/100-days-of-oss.md:47]: [Addition] I think there should be a section on balancing this challenge here. One of the criticisms I heard about 100 Days of Code is that implies this needs to be done for 100 days straight,so putting something like "take breaks" or "reward yourself for doing something OSS-related" would help students to participate.

File: docs/community/100-days-of-oss.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- **Take our Intro To Open Source Course:** If you're new to open source, we recommend taking our [Intro to Open Source](https://github.com/open-sauced/intro) course to learn more about open source and how to get started.
+
+## The Power of the #100DaysOfOSS Challenge
+The #100DaysOfOSS challenge offers a supportive community where developers can find encouragement, share experiences, and overcome roadblocks together. 
+
+Our hope is that the community will provide a safe space to discuss challenges, celebrate achievements, and exchange insights, creating an environment that helps individuals stay on track and avoid giving up.
+
+
+## Why join the #100DaysOfOSS Challenge:
+
```
Comment (CBID2) [inline docs/community/100-days-of-oss.md:60]: [Nitpick] I'd mention something like tuition-fee free. It’ll pique new contributor’s interest.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @BekahHW! I know I gave you my review via DMs, but I found the PR, so I decided to give more feedback. I hope you find my suggestions helpful. 😃

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**